src/query/service/src/interpreters/interpreter_factory_interceptor.rs

```diff
@@ -75,11 +75,8 @@ impl Interpreter for InterceptorInterpreter {
     }
 
     async fn execute(&self, ctx: Arc<QueryContext>) -> Result<SendableDataBlockStream> {
-        // Management mode access check.
-        match &self.new_plan {
-            Some(p) => self.management_mode_access.check_new(p)?,
-            _ => self.management_mode_access.check(&self.plan)?,
-        }
+        // Access check.
+        self.accessor_checker.check(&self.new_plan, &self.plan)?;
```

Comment:
Maybe move to InterpreterFactory is better?

Reply:
> Maybe move to InterpreterFactory is better? 
Looks better to add them to `create_interpreter`.

>  We can ignore queries without permission in the query_log.

Does it mean we don't log the error query in the query_log?